limDaSeul/app.js

```diff
@@ -49,6 +49,124 @@ app.post("/users", async (req, res, next) => {
   res.status(201).json({message : "userCreated"});
 })
 
+//Create post table
+app.post("/posts", async(req, res, next) => {
+  const { title, content, posting_image, user_id } = req.body
+
+  await appDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      posting_image,
+      user_id
+    ) VALUES(?, ?, ?, ?);
+    `,
+    [title, content, posting_image, user_id]
+  );
+  res.status(200).json({message : "postCreated"});
+})
+
+//Get post list
+app.get("/posts", async(req, res) => {
+
+  await appDataSource.manager.query(
+    `SELECT
+        u.profile_image as userProfileImage,
+        u.id as userID,
+        p.id as postingId,
+        p.posting_image as postingImageUrl,
+        p.content as postingContent
+      FROM users u, posts p
+      WHERE u.id = p.id
+      `,
+      (err, rows) =>  {
+         res.status(200).json(rows);
+    })
+})
+
+//Update post
+app.patch('/posts/update_des/:postID', async(req, res) => {
+  const postID = req.params.postID;
+  const {title, content, posting_image} = req.body
+
+  await appDataSource.query(
+    `UPDATE posts
+    SET
+      title = ?,
+      content = ?,
+      posting_image = ?
+    WHERE id = ${postID}
+   `,
+    [title, content, posting_image]
+    );
+
+  const posting = await appDataSource.query(
+    `SELECT
+        u.id as userID,
+        u.name as userName,
+        p.id as postingId,
+        p.title as postingTitle,
+        p.content as postingContent
+      FROM posts p
+      INNER JOIN users u
+      ON u.id = p.user_id
+      WHERE p.id
+    `);
+  res.status(201).json({data : posting[0]});
+});
+
+//Delete post
+app.delete('/posts/:postID', async(req, res) => {
+  const {postID} = req.params;
+
+  await appDataSource.query(
+    `DELETE FROM posts
+    WHERE posts.id = ${postID}
+    `);
+      res.status(200).json({message : "postingDeleted"});
+})
+
+//Update user's 'like it'
+app.post('/likelist', async(req, res) => {
+  const {user_id, postID} = req.body
+
+  await appDataSource.query(
+    `INSERT INTO likelist(
+      user_id,
+      postID
+    )VALUES(?,?);
+    `,
+    [user_id, postID]
+  );
+
+  res.status(200).json({message : "likeCreated"});
+})
```

Comment:
주석과 해당 코드만 보았을 때 일치하지 않아보입니다.

주석 삭제해주시고 엔드포인트도 수정해주세요!
```suggestion
//Update user's 'like it'
app.post('/likes', async(req, res) => {
  const {user_id, postID} = req.body

  await appDataSource.query(
    `INSERT INTO likelist(
      user_id,
      postID
    )VALUES(?,?);
    `,
    [user_id, postID]
  );

  res.status(200).json({message : "likeCreated"});
})
```